### DISCUSSION TITLE
MSIX packages in GitHub releases and winget

### DISCUSSION THREAD

**Discussion by just1a-person** [PowerShell/PowerShell#14803] [General]
I couldn't find MSIX packages in GitHub releases. It would be great if you could publish them. Also please include those in winget so that I can update it by:

```
winget upgrade powershell
```

**Comment by SteveL-MSFT** [comment]
Is there a specific reason you would want to manually install the msix outside of the MS Store?  Side-loading requires Developer Mode (I believe) which limits it's usefulness for most users.

**Reply by just1a-person** [reply]
> Side-loading requires Developer Mode (I believe)

I don't think it requires Developer Mode. Even Terminal provides MSIX releases. and WinGet wouldn't have accepted MSIX packages if it required Developer Mode

> Is there a specific reason you would want to manually install the msix outside of the MS Store? 

Better install and upgrade experience for those who installed PowerShell using winget

**Reply by just1a-person** [reply]
Sorry for pinging @SteveL-MSFT

**Comment by SteveL-MSFT** [comment]
winget can already install PowerShell and it appears that the `upgrade` functionality isn't complete in the winget client yet, but I expect it to work with our MSI package

**Reply by just1a-person** [reply]
I installed PowerShell using [this manifest](https://github.com/microsoft/winget-pkgs/tree/master/manifests/Microsoft/powershell-preview-msix).

It is not updated for newer versions.

**Reply by just1a-person** [reply]
@SteveL-MSFT I believe that the `upgrade` command works with only MSIX packages

**Comment by roke-julian-lockhart** [comment]
https://github.com/microsoft/winget-pkgs/discussions/275#discussion-15755
